modular_skyrat/modules/goofsec/code/sec_clothing_overrides.dm

```diff
@@ -787,6 +787,35 @@
 		/obj/item/gun/ballistic/rifle/boltaction, //fits if you make it an obrez
 		))
 
+/obj/item/storage/belt/holster/chameleon
```

Comment:
did you know that syndicate holsters in suit slots probably can break really bad due to the fact that not every belt has a belt_mirror sprite, which suit slot items use. I'm pretty sure I blocked syndie holsters from fitting in normal armor vests for this reason.